openapi/java.xml

```diff
@@ -0,0 +1,66 @@
+<project 
+  xmlns="http://maven.apache.org/POM/4.0.0" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.kubernetes</groupId>
+  <artifactId>client-java</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>client-net</name>
+  <url>http://kubernetes.io</url>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger</groupId>
+        <artifactId>swagger-codegen-maven-plugin</artifactId>
+        <version>2.2.2</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${generator.spec.path}</inputSpec>
+              <language>java</language>
+              <gitUserId>kubernetes-incubator</gitUserId>
+              <gitRepoId>client-java</gitRepoId>
+              <configOptions>
```

Comment:
These settings should be supported for java client generation. These are the valid settings:

```bash
$ swagger-codegen config-help -l java

CONFIG OPTIONS
	sortParamsByRequiredFlag
	    Sort method arguments to place required parameters before optional parameters. (Default: true)

	ensureUniqueParams
	    Whether to ensure parameter names are unique in an operation (rename parameters that are not). (Default: true)

	modelPackage
	    package for generated models

	apiPackage
	    package for generated api classes

	invokerPackage
	    root package for generated code

	groupId
	    groupId in generated pom.xml

	artifactId
	    artifactId in generated pom.xml

	artifactVersion
	    artifact version in generated pom.xml

	sourceFolder
	    source folder for generated code

	localVariablePrefix
	    prefix for generated code members and local variables

	serializableModel
	    boolean - toggle "implements Serializable" for generated models (Default: false)

	bigDecimalAsString
	    Treat BigDecimal values as Strings to avoid precision loss. (Default: false)

	fullJavaUtil
	    whether to use fully qualified name for classes under java.util. This option only works for Java API client (Default: false)

	hideGenerationTimestamp
	    hides the timestamp when files were generated

	dateLibrary
	    Option. Date library to use
	        joda - Joda
	        legacy - Legacy java.util.Date
	        java8-localdatetime - Java 8 using LocalDateTime (for legacy app only)
	        java8 - Java 8 native

	useRxJava
	    Whether to use the RxJava adapter with the retrofit2 library. (Default: false)

	library
	    library template (sub-template) to use (Default: okhttp-gson)
	        jersey1 - HTTP client: Jersey client 1.19.1. JSON processing: Jackson 2.7.0
	        feign - HTTP client: Netflix Feign 8.16.0. JSON processing: Jackson 2.7.0
	        jersey2 - HTTP client: Jersey client 2.22.2. JSON processing: Jackson 2.7.0
	        okhttp-gson - HTTP client: OkHttp 2.7.5. JSON processing: Gson 2.6.2
	        retrofit - HTTP client: OkHttp 2.7.5. JSON processing: Gson 2.3.1 (Retrofit 1.9.0). IMPORTANT NOTE: retrofit1.x is no longer actively maintained so please upgrade to 'retrofit2' instead.
	        retrofit2 - HTTP client: OkHttp 3.2.0. JSON processing: Gson 2.6.1 (Retrofit 2.0.2). Enable the RxJava adapter using '-DuseRxJava=true'. (RxJava 1.1.3)

```